src/certgen_certificate_generator_server.cc

```diff
@@ -27,11 +27,285 @@
 */
 
 #include "fty_certificate_generator_classes.h"
+#include <chrono>
+#include <netdb.h>
+
+// because there is NO std::chrono::days in C++14
+#define HOURS_IN_DAY        24
+
+#define COMMON_NAME         "Web server-ipm2-jana-selfsigned"
```

Comment:
CN need to be created at runtime => <config name>-<hostname>[-selfsigned]

Reply:
Addressed.